src/App.css

```diff
@@ -1,38 +1,50 @@
-.App {
-  text-align: center;
+html,
+body {
```

Comment:
use this to reset css
https://meyerweb.com/eric/tools/css/reset/

Reply:
i will